integration-cli/docker_api_images_test.go

```diff
@@ -93,7 +93,7 @@ func (s *DockerSuite) TestAPIImagesDelete(c *check.C) {
 	c.Assert(err.Error(), checker.Contains, "unable to delete")
 
 	_, err = cli.ImageRemove(context.Background(), "test:noexist", types.ImageRemoveOptions{})
-	c.Assert(err.Error(), checker.Contains, "No such image")
+	c.Assert(err.Error(), checker.Contains, "no such image")
 
```

Comment:
Might be usefull to check with client.IsNotFound(err) here (and do that for every api/remove tests)

Reply:
This is already done in the `client/` unit tests, which I think it the appropriate place to do this. The API tests should be testing the API not the client.